scripts/pre-commit

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+if ! command -v black &> /dev/null
+then
+  echo 'ERROR: `black` not found. Is your virtualenv active?'
+  exit 1
+fi
+
+black . --check
```

Comment:
Ignoreekos tämä muuten venvin jos venv on samassa hakemistossa? Periaateessa sen alla kai saattais olla python filuja joita ei kyllä haluta formatoida. Ja venvin laittaminen samaan hakemistoon projektin juuren kanssa tuntuu olevan aika yleistä. Tai no ainakin ite teen niin.

Reply:
Koodin formatoinnille voisi myös lisätä invoke taskin niin saisi sen ajettua käsinkin tarvittaessa projektin task-runnerin kautta. Periaatteessa sinne voisi konffata ehkä tuon blackin ajamisen mahdollisine exclude/includeasetuksineen ja sitten täällä kutsuisi invokea...?

Reply:
> Ignoreekos tämä muuten venvin jos venv on samassa hakemistossa?
Defaulttina ignore noudattaa `.gitignore`a :+1: 

Unohdin koko `invoke`n olemassaolon :sweat_smile:  Teen näin, kiitos :sparkles: